packages/worker/src/sdk/users/users.ts

```diff
@@ -77,6 +79,9 @@ export const paginatedUsers = async ({
   } else if (email) {
     userList = await usersCore.searchGlobalUsersByEmail(email, opts)
     property = "email"
+  }
+  if (userIds) {
+    // TODO: search users by userIds
```

Comment:
I think we need to implement this for the backups-ui to be able to search for the users associated with the backups loaded via the `createdBy` field

Reply:
Ah I'll remove this - we don't need to do this because its done on the backend, enriched instead, much more efficient that way.

Reply:
Yep far better, good shout 👍